src/backend/distributed/commands/index.c

```diff
@@ -597,8 +601,11 @@ PreprocessReindexStmt(Node *node, const char *reindexCommand,
 									   "tables are not supported")));
 			}
 
+			ObjectAddress relationAddress = { 0 };
+			ObjectAddressSet(relationAddress, RelationRelationId, relationId);
+
 			DDLJob *ddlJob = palloc0(sizeof(DDLJob));
-			ddlJob->targetRelationId = relationId;
+			ddlJob->targetObjectAddress = relationAddress;
```

Comment:
can we avoid the additional variable and do:
```C
ObjectAddressSet(ddlJob->targetObjectAddress, RelationRelationId, relationId);
```

Because it seems to add non-negligible amount of repetition?